docs/docs/learn/1_why_aztec/1_privacy_problem.md

```diff
@@ -5,46 +5,58 @@ description: "Understanding why privacy matters in blockchain, how Web2 handles
 
 ## Why Aztec?
 
-Welcome to one of the most important topics in blockchain technology - privacy.
+Welcome to one of the most important topics in blockchain technology: privacy.
 
 - [TODO] add point about how important financial privacy is
 
-Let's explore why privacy is such a critical challenge in blockchain, and more importantly, why solving it opens up incredible opportunities for innovation.
+Let's explore why privacy is such a critical challenge for blockchains, and more importantly, why solving it opens up opportunities for innovation.
 
 ## Learning Objectives
 
 By the end of this section, you'll understand:
 
 1. Why privacy matters in blockchain
 2. Why Web2 is often better for privacy (surprising, right?)
-3. The exciting opportunities that private applications enable
+3. The opportunities that private applications enable
 
 ---
 
 ## Why Privacy Matters in Blockchain
 
-### The Transparency Paradox
+### Problem 1: Blockchain Transparency
 
-Here's something that might surprise you: blockchain's greatest strength - transparency - is also one of its biggest weaknesses when it comes to privacy.
+Blockchains are transparent by design. This is what makes them so useful: no one can act dishonestly because every transaction is visible on-chain.
```

Comment:
```suggestion
Blockchains are transparent by design. This is what makes them so useful: no one can act dishonestly because every transaction is visible onchain.
```
we don't hyphenate onchain for whatever reason :slightly_smiling_face: